module/account/features/account-token.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Account Token
+
+  Scenario: Reset Token user
+    And I add "Content-Type" header equal to "application/json"
+    And I add "Accept" header equal to "application/json"
+    When I send a POST request to "/api/v1/accounts/token/generate" with body:
+      """
+      {
+          "email": "test@ergonode.com",
+          "path": "http://localhost/rest?token="
+      }
+      """
```

Comment:
Is it possible to verify whether the mail has been passed to be sent?

Reply:
I don't know, I'll check in the next CR